README.md

```diff
@@ -1,32 +1,12 @@
-<a id="markdown-transportd---http-middleware-as-a-service" name="transportd---http-middleware-as-a-service"></a>
 # transportd - HTTP Middleware As A Service
 [![GoDoc](https://godoc.org/github.com/asecurityteam/transportd?status.svg)](https://godoc.org/github.com/asecurityteam/transportd)
 [![Build Status](https://travis-ci.com/asecurityteam/transportd.png?branch=master)](https://travis-ci.com/asecurityteam/transportd)
 [![codecov.io](https://codecov.io/github/asecurityteam/transportd/coverage.svg?branch=master)](https://codecov.io/github/asecurityteam/transportd?branch=master)
 
 *Status: Incubation*
 
-<!-- TOC -->
-
-- [transportd - HTTP Middleware As A Service](#transportd---http-middleware-as-a-service)
-    - [Overview](#overview)
-    - [Configuration](#configuration)
-        - [Runtime Settings](#runtime-settings)
-        - [Backend Settings](#backend-settings)
-        - [Route Settings](#route-settings)
-        - [Environment Variables](#environment-variables)
-    - [Custom Plugins And Builds](#custom-plugins-and-builds)
-        - [Custom Components](#custom-components)
-        - [Writing A Component](#writing-a-component)
-        - [Generating A Build](#generating-a-build)
-    - [Using As A Library](#using-as-a-library)
-    - [Contributing](#contributing)
-        - [License](#license)
-        - [Contributing Agreement](#contributing-agreement)
-
-<!-- /TOC -->
-
-<a id="markdown-overview" name="overview"></a>
+<!-- TOC -->autoauto- [transportd - HTTP Middleware As A Service](#transportd---http-middleware-as-a-service)auto    - [Overview](#overview)auto    - [Configuration](#configuration)auto        - [Runtime Settings](#runtime-settings)auto        - [Backend Settings](#backend-settings)auto        - [Route Settings](#route-settings)auto        - [Environment Variables](#environment-variables)auto    - [Custom Plugins And Builds](#custom-plugins-and-builds)auto        - [Custom Components](#custom-components)auto        - [Writing A Component](#writing-a-component)auto        - [Generating A Build](#generating-a-build)auto    - [Using As A Library](#using-as-a-library)auto    - [Contributing](#contributing)auto        - [License](#license)auto        - [Contributing Agreement](#contributing-agreement)autoauto<!-- /TOC -->
```

Comment:
task: Looks like the newlines from this were removed and all the anchor points below are gone.

Reply:
Huh.  I'll fix.